pygotham/sponsors/models.py

```diff
@@ -56,6 +56,7 @@ class Sponsor(db.Model):
     contact_email = db.Column(db.String(255))
     accepted = db.Column(db.Boolean)
     payment_received = db.Column(db.Boolean)
+    twitter_handle = db.Column(db.String(255))
```

Comment:
We should use the same type and constraints [used elsewhere](https://github.com/PyGotham/pygotham/blob/90193277c175eecfd49e9514e971484d035d3526/pygotham/users/models.py#L67).